service/iot-service-samples/device-twin-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceTwinSample.java

```diff
@@ -152,14 +154,16 @@ private static void cancelJob(DeviceTwin twinClient, DeviceTwinDevice device) th
         desiredProperties.add(new Pair("temp", new Random().nextInt(TEMPERATURE_RANGE)));
         desiredProperties.add(new Pair("hum", new Random().nextInt(HUMIDITY_RANGE)));
         device.setDesiredProperties(desiredProperties);
+        // ScheduleUpdateTwin job type is a force update, which only accepts '*' as the Etag
+        device.setETag("*");
 
         // date when the update shall be executed
         Date updateDateInFuture = new Date(new Date().getTime() + ADD_10_MINUTES_IN_MILLISECONDS); // 10 minutes in the future.
 
         // query condition that defines the list of device to be updated
         String queryCondition = "DeviceId IN ['" + deviceId + "']";
 
-        System.out.println("Schedule updating Device twin (new temp, hum) in 10 minutes");
+        System.out.println("Cancel updating Device twin (new temp, hum) in 10 minutes");
```

Comment:
Is this correct though? I see schedule update call next to this line

Reply:
It is the same method but our sample is using it to cancel. First time we use it to schedule and second time to cancel. Our logs should indicate that and not that we are calling the same function.